Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:18.04
```

Comment:
[Q] Which Ubuntu version is installed on CI machines? Is it safe to run ubuntu 20 docker on ubuntu 18 machine?

Reply:
That's indeed a TODO item after changes in this PR looks stable for the CI: we need to update CI machines to 20.04.